source/includes/fact-match-dump-restore-versions.rst

```diff
@@ -1,9 +1,26 @@
 When using :binary:`~bin.mongorestore` to load data files created by
-:binary:`~bin.mongodump`, be sure that you are restoring to the *same
-major version* of the MongoDB Server that the files were created from.
-For example, if your dump was created from a MongoDB Server running
-version ``4.4.x``, be sure that the MongoDB Server you are restoring to
-is also running version ``4.4.x``.
+:binary:`~bin.mongodump`, be sure that the deployment you are restoring 
+to has either:
+
+- The same major version.
+- The same feature compatibility version. 
+
+For example, if your dump was created from a MongoDB deployment running
+version ``4.4``, be sure that the MongoDB deployment you are restoring 
```

Comment:
Similar to my comment above, I'm not a huge fan of "be sure" in this context. I think stating these as things which "must" be true would be more direct.

Consider:

"... from a MongoDB deployment running version ``4.4``, the MongoDB deployment you restore to must run also version ``4.4`` or have its FCV level set to ``4.4``.